backend/app.py

```diff
@@ -6,62 +6,63 @@
 - jq python api docs: https://github.com/mwilliamson/jq.py
 """
 import json
-import os.path  # TODO: Siggie confused: are you supposed to use path.<method>(...) or os.path.<method>(...)?
```

Comment:
@Sigfried I mentioned this once before but IDK if you saw. I always thought this was strange before. The typical pattern is `from os import path`, and then I would be doing like `my_path = path.join(...)`. But I've gotten warnings before. Idk, maybe this is solved in a newer version of Python. I should double check.

Reply:
Hmm I got no warning in my IDE. maybe I should change it all to `from os import path`